decidim-admin/app/views/decidim/admin/logs/index.html.erb

```diff
@@ -1,3 +1,4 @@
+<% add_decidim_page_title(t("admin_log", scope: "decidim.admin.menu")) %>
```

Comment:
Better to use it with "titles" instead of "menu", so it is "Admin log" (as the view title) instead of "Admin activity log" (as the menu/sidebar) 
```suggestion
<% add_decidim_page_title(t("admin_log", scope: "decidim.admin.titles")) %>
```